src/Components/Navbar/Navbar.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Link } from "react-router-dom"
+import NavLogo from "../../Assets/nav-logo.png"
+
+const Navbar = () => {
+    return (
+        <nav class="navbar">
+            <div class="nav-logo">
+                <img src={NavLogo} alt="logo" />
```

Comment:
Suggestion: I think you forgot to change `class` to `className`

Reply:
done